yo1k/sandbox/exercises/CSC_algorithms/test_sorting.py

```diff
@@ -0,0 +1,54 @@
+import itertools
+from collections.abc import Sequence, MutableSequence
+from typing import Any
+
+import pytest
+
+import yo1k.sandbox.exercises.CSC_algorithms.sorting as my_sort
+
+
+@pytest.fixture
+def arrays_set() -> Sequence[MutableSequence[Any]]:
+    return (
+            [1, 2, 5, 7, 3, 10, 4, -1, 5],
```

Comment:
Why are you calling everything "array" both in tests and in exercises? I thought these things are lists, and at places where you don't know that the objects are lists, you could call them sequences (`seq` may be a short name).

Reply:
Done.